src/bin/pred.rs

```diff
@@ -73,6 +77,15 @@ define_language! {
   }
 }
 
+/// Return a non-zero constant.
```

Comment:
nit: technically could be 0 right? We don't check that `n` is non-zero

Reply:
good catchm I meant denominator, will fix.